data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/KafkaSourceConfig.java

```diff
@@ -25,10 +26,28 @@ public class KafkaSourceConfig {
   @Size(min = 1, message = "Bootstrap servers can't be empty")
   private List<String> bootStrapServers;
 
-  @JsonProperty("topics")
+    @JsonProperty("topics")
```

Comment:
minor: indentation looks off.

Reply:
Please address these in Bug : https://github.com/opensearch-project/data-prepper/issues/2713